spec/models/reservation_spec.rb

```diff
@@ -494,6 +494,39 @@
     end
   end
 
+  context 'when manually over-extending reservation' do
+    before(:each) do
+      @em = FactoryGirl.create(:equipment_model)
+      @ei = FactoryGirl.create(:equipment_item, equipment_model: @em)
+
+      FactoryGirl.create(:valid_reservation)
```

Comment:
You're not assigning this any variable (so you're not actually updating its parameters in the 4 lines below and in the final `it` block). You can specify attributes in the `FactoryGirl#create` call directly as well:

```
FactoryGirl.create(:valid_reservation, equipment_model: @em, equipment_item: @ei, start_date: Time.zone.today + 1.day, due_date: Time.zone.today + 2.days)
```

Finally, `reservation` is actually defined at the [top of this file](https://github.com/YaleSTC/reservations/blob/1058_check_over_reserving_editing_reservations_2/spec/models/reservation_spec.rb#L24) as a built valid reservation for some random equipment model with a single equipment item, so that's what you're updating in the next four lines. Granted, it almost fulfills the original purpose except for the fact that `reservation` hasn't yet been saved to the database so you're not actually testing your changes. You can just delete this line and run `reservation.assign_attributes(equipment_item: @ei, ...)` or even `update_attributes` to save the `reservation` object with the desired changes in one call.